cli/src/trusted_commands.rs

```diff
@@ -540,3 +622,134 @@ fn unshield_funds(
 			.into_trusted_operation(trusted_args.direct);
 	let _ = perform_operation(cli, trusted_args, &top);
 }
+
+#[cfg(feature = "evm")]
+fn evm_create(cli: &Cli, trusted_args: &TrustedArgs, arg_from: &str, smart_contract: &str) {
+	let from = get_pair_from_str(trusted_args, arg_from);
+	let from_acc: AccountId = from.public().into();
+	println!("from ss58 is {}", from.public().to_ss58check());
+
+	let mut sender_evm_acc_slice: [u8; 20] = [0; 20];
+	sender_evm_acc_slice.copy_from_slice((<[u8; 32]>::from(from_acc.clone())).get(0..20).unwrap());
+	let sender_evm_acc: H160 = sender_evm_acc_slice.into();
+
+	let (mrenclave, shard) = get_identifiers(trusted_args);
+
+	let nonce = get_layer_two_nonce!(from, cli, trusted_args);
+
+	let sender_evm_substrate_addr =
+		HashedAddressMapping::<BlakeTwo256>::into_account_id(sender_evm_acc);
+	println!("Trying to get nonce of evm account {:?}", sender_evm_substrate_addr.to_ss58check());
+	let evm_account_nonce = get_layer_two_evm_nonce!(from, cli, trusted_args);
+
+	let top = TrustedCall::evm_create(
+		from_acc,
+		sender_evm_acc,
+		Vec::from_hex(smart_contract.to_string()).unwrap(),
+		U256::from(0),
+		967295,        // gas limit
+		U256::from(1), // max_fee_per_gas !>= min_gas_price defined in runtime
+		None,
+		None,
+		Vec::new(),
+	)
+	.sign(&from.into(), nonce, &mrenclave, &shard)
+	.into_trusted_operation(trusted_args.direct);
+
+	let _ = perform_operation(cli, trusted_args, &top);
+
+	let execution_address = evm_create_address(sender_evm_acc, evm_account_nonce);
+	info!("trusted call evm_create executed");
+	println!("Hopefully created the smart contract with address {:?}", execution_address);
```

Comment:
As much as this is true, this is not nice in a demo. (The "Hopefully" I mean). 

We have two options here:
a) Remove the Hopefully and just make it a statement, even though this might not be true
b) Do a getter and check if the return value is Some (just like in the unit  tests) and only then print the success message